pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -1092,6 +1092,15 @@ typedef struct pjsua_call_setting
      */
     pjmedia_dir      media_dir[PJMEDIA_MAX_SDP_MEDIA];
 
+    /**
+     * Custom Call-ID to be sent out with outgoing INVITE. Overrides default
+     * Call-ID generated by dialog.
+     *
+     * This setting will only be used when creating a new outgoing call via pjsua_call_make_call().
```

Comment:
- Better add info/warning that the library won't verify its validity (e.g: allowed chars, uniqueness).
- Column width should be < 80, for both, code & comment.
  Rather old school, but for consistency :)

Reply:
ok sounds good :)